frontend/src/views/mspcontracts/UpdateContractAddress.vue

```diff
@@ -187,8 +196,17 @@ export default {
     resetForm() {
       this.groupNumber = ''
       this.phn = ''
-      this.homeAddress = {}
-      this.mailingAddress = {}
+      this.telephone = ''
+      this.homeAddress.addressLine1 = ''
+      this.homeAddress.addressLine2 = ''
+      this.homeAddress.addressLine3 = ''
+      this.homeAddress.addressLine4 = ''
+      this.homeAddress.postalCode = ''
+      this.mailingAddress.addressLine1 = ''
+      this.mailingAddress.addressLine2 = ''
+      this.mailingAddress.addressLine3 = ''
+      this.mailingAddress.addressLine4 = ''
+      this.mailingAddress.postalCode = ''
```

Comment:
Do we need to do this level of detailed reset? I think we can just set to an empty object. But please verify.

Reply:
Setting entire address object as empty caused all the address field to become 'undefined' leading mailing address required if validation failed. Working fine with detailed reset.